Jakefile

```diff
@@ -1191,7 +850,7 @@ task('_build-file', { async: true }, function() {
       ].join(' ')
     ], EXEC_OPTS);
     _getCommintHash.on('stdout', function(COMMIT_INFO) {
-        var BUILD_TIME = _extended_build_time(),
+        var BUILD_TIME = _extended_build_time();
```

Comment:
uh, my common mistake. then please remove an indentation here and move it a line above, so it will be visually in the other place for me, not to fix it _improperly_ :).